week1/1-2/index.html

```diff
@@ -0,0 +1,105 @@
+                    </div>
+                    <div>
+                        <input type="checkbox" name="out" checked>
+                        <label for="out">지출</label>
+                    </div>
+                </div>
+            </div>
+            <div class="history_list">
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">식비</p>
+                    <h3 class="history_title">공차경기대후문점</h3>
+                    <p class="history_asset minus">-9000</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">생필품</p>
+                    <h3 class="history_title">세븐일레븐</h3>
+                    <p class="history_asset minus">-1100</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">월급</p>
+                    <h3 class="history_title">알바 월급</h3>
+                    <p class="history_asset plus">+400000</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">쇼핑</p>
+                    <h3 class="history_title">흰색 신발 구매</h3>
+                    <p class="history_asset minus">-100000</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">취미</p>
+                    <h3 class="history_title">포토시그니처</h3>
+                    <p class="history_asset minus">-5000</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">취미</p>
+                    <h3 class="history_title">볼링</h3>
+                    <p class="history_asset plus">-9500</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">취미</p>
+                    <h3 class="history_title">반지공방</h3>
+                    <p class="history_asset minus">-95000</p>
+                </div>
+                <div class="history_item">
+                    <img src="assets/x_icon.png" alt="x 버튼" class="x_icon">
+                    <p class="history_category">쇼핑</p>
+                    <h3 class="history_title">옷 구매</h3>
+                    <p class="history_asset plus">-45000</p>
+                </div>
+            </div>
+        </div>
+    </div>
+    <footer>
+        <img src="./assets/add_icon.png" alt="">
```

Comment:
마찬가지로
```suggestion
        <button  type="button"><img src="./assets/add_icon.png" alt=""></button>
```
그리고 alt 써주깅 ~!